README.md

```diff
@@ -69,4 +69,21 @@ You can get official support from DJI and the community with the following metho
 You can also communicate with other developers by the following methods:
 
 - Post questions on [**Stackoverflow**](http://stackoverflow.com) using [**
-  dji-sdk**](http://stackoverflow.com/questions/tagged/dji-sdk) tag
\ No newline at end of file
+  dji-sdk**](http://stackoverflow.com/questions/tagged/dji-sdk) tag
+
+
+## Packaging a `debian` package 
+Below are simple instructions to generate a `.deb` package for this repo that can be distributed using apt.
+To use this package, first install it with `dpkg -i <nameOfDeb>.deb`. 
+In `CMakeLists.txt` do `find_package(PayloadSDK)` and link any targets with the appropriate  
+`target_include_directories(myTarget ${PayloadSDK_INCLUDE_DIRS})` and executables or libraries with
+`target_link_libraries(myTarget ${PayloadSDK_LIBRARIES})` 
+**Steps**
```

Comment:
will these steps be added in the circleCI to auto generate the debian package in cloudsmith?

Reply:
They'll be added in eventually, as there are some things I need to figure out first for versioning and CI. 

For right now, manual package creation and upload should be sufficient, and will form the basis for automated distribution, especially since we won't be updating this external dependency often.